ds3-sdk/src/main/java/com/spectralogic/ds3client/commands/interfaces/AbstractResponse.java

```diff
@@ -154,6 +154,18 @@ private String readResponseString() throws IOException {
         }
     }
 
+    protected Integer parseIntHeader(final String key) {
+        final List<String> list = getResponse().getHeaders().get(key);
+        switch (list.size()) {
+            case 0:
+                return null;
+            case 1:
+                return Integer.parseInt(list.get(0));
+            default:
+                throw new IllegalArgumentException("Response has more than one header value for " + key);
```

Comment:
There is no test for this function directly. To my knowledge, there is no instance where the BP should return multiple pagination headers (where this is used), so I can't create an integration test.  Should I add a unit test?